tests/test_observation.py

```
@@ -163,12 +163,12 @@ async def __aenter__(self):
                 self._conn = await self._real.__aenter__()
                 original_execute = self._conn.execute
 
-                def patched_execute(sql, *a, **kw):
+                async def patched_execute(sql, *a, **kw):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `patched_execute`

(ANN202)